src/pages/dashboard/faq.tsx

```
@@ -0,0 +1,248 @@
+                    <AccordionItem
+                      className="my-5 rounded-lg border border-slate-200 bg-white"
+                      key={faq.question}
+                      value={faq.question}
+                    >
+                      <AccordionTrigger className="rounded px-5 py-5 text-base font-normal hover:bg-slate-200/5 hover:no-underline">
+                        <span className="flex-1 text-left text-lg font-medium">
+                          {faq.question}
+                        </span>
+                      </AccordionTrigger>
+                      <AccordionContent className="px-5 pb-6 text-base text-slate-700">
+                        <div dangerouslySetInnerHTML={{ __html: faq.answer }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)